File: ejercicio2.java

import java.util.Scanner;

public class ejercicio2 {

	public static void main(String[] args) {
	
	System.out.println("Que pasa tu, como te llamas?");
	
	Scanner teclado = new Scanner(System.in);
	String nombre;
	nombre = teclado.nextLine();
	System.out.println("\nEncantado de conocerte, que tal " +nombre+"\n");
	
	
	Scanner teclado2 = new Scanner(System.in);
	int edad;
	System.out.println("Cuantos anyos tienes "+nombre+"?");
	edad = teclado2.nextInt();
	
	int suma;
	suma=(edad+10);
	System.out.println("\n"+"Dentro de 10 anyos tendras "+(edad+10)+" anyos, don "+nombre);
	

	}
}

File: ejercicio2.cpp
int main(int argc, char const *argv[])
{
	string nombre;
	int edad;
	cout<<"Bienvenido usuario cual es vuestro nombre?"<<endln;
	cin>>nombre>>endln;
	cout<<"Encantado de conocerte "<<nombre<<" cual es vuestra edad?"<<endln;
	cin>>edad>>endln;
	cout<<"Tendras "<<(edad + 10)<<" dentro de 10 años."<<endln;
	return 0;
}